tests/unit_tests/getAllFields.test.js

```diff
@@ -122,13 +119,23 @@ it(`creates the right data for a 'choices:' field`, async function() {
 // ```
 // - Something: some_var
 //   datatype: object
-//   object labeler: |
-//     lambda y: y.short_label()```
-//   choices: some_obj
+//   choices:
+//     - obj1
+//     - obj2
 // ```
-it(`creates the right data for a 'choices:' field`, async function() {
-  // `field:` and `choices:`
+it(`creates the right data for a dropdown created with an object`, async function() {
   let result = await getAllFields( scope, { html: html.object_dropdown });
   expect( result ).to.deep.equal( fields.object_dropdown );
+});
+
+
+// ============================
+// DOM for mixed quotes test in getMatchingRows.test.js
+// ============================
+// Helps us create the mixed quotes test in getMatchingRows.test.js, but
+// commented out because it's not a useful test for getAllFields()
+it(`creates the right data for a question whose table will have mixed quotes`, async function() {
```

Comment:
The comment in the code is a bit confusing, so thanks for pointing it out. The log statement is incidental and was for debugging.

This is the deal: The code here does double duty. It does test `.getAllFields()`, but those tests also use actual docassemble HTML to help create the objects in the fields.fixtures.js files which then help with the `.getMatchingRows()` tests. This particular test is solely for that purpose. The console log in there is what I've used to get that fields.fixtures.js object, but it can be removed in between runs.

I'll try to write the comment so it has more context.